config/puma.rb

```diff
@@ -9,15 +9,12 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 3000 }
+port        ENV.fetch("PORT") { 4004 }
```

Comment:
This _could_ be overridden in the deploy, if you want to keep the conventional 3000 for dev work.

Reply:
i use port 3000 by default when debugging in rubymine.  can this be overridden by the deploy and keep the convention of 3000 for local dev?

Reply:
Yup, will revert.

Reply:
reverted

Reply:
(also added matching override to deployment)